test/specs/controller.scatter.test.js

```diff
@@ -3,6 +3,28 @@ describe('Chart.controllers.scatter', function() {
 		expect(typeof Chart.controllers.scatter).toBe('function');
 	});
 
+	it('should test default tooltip callbacks', function() {
+		var chart = window.acquireChart({
+			type: 'scatter',
+			data: {
+				datasets: [{
+					data: [{
+						x: 10,
+						y: 15
+					}],
+					label: 'dataset1'
+				}],
+			},
+			options: {}
+		});
+		var point = chart.getDatasetMeta(0).data[0];
+		jasmine.triggerMouseEvent(chart, 'mousemove', point);
+
+		// Title should be empty
```

Comment:
[nit] it looks like the spacing here is different than the line below

Reply:
You mean between the line 'Title should be empty' and the expect lines?
Could you point me to what the difference is? They look the same offset between the 2 lines.

Using `cat -T test/specs/controller.scatter.test.js` I see the same number of tabs shown as `^I` when using `cat -T`
```
^I^I// Title should be empty
^I^Iexpect(chart.tooltip._view.title.length).toBe(0);
^I^Iexpect(chart.tooltip._view.body[0].lines).toEqual(['(10, 15)']);
```
If I'm missing something let me know. Thanks.

Reply:
Sorry, this must have been a mobile glitch. I was reviewing on iOS and it showed the comment line as if it only had a single tab. When I look now on the desktop UI it looks fine so there's nothing to fix